### DISCUSSION TITLE
Population Set Attribute Error

### DISCUSSION THREAD

**Discussion by rabiu42000** [anyoptimization/pymoo#215] [Q&A] [answered]
I get this error. Any ideas on what is causing it?

pymoo\core\population.py", line 58, in set
    raise Exception("Population Set Attribute Error: Number of values and population size do not match!")

**Comment by blankjul** [comment, marked as answer]
In `_evaluate` the array to set to `out` is most likely not in the correct shape. If you use `ElementwiseProblem` it should be for `out["F"]` a vector of length `problem.n_obj`.
